contrib/common/apt/ralph.list

```diff
@@ -1 +1 @@
-deb https://dl.bintray.com/vi4m/ralph bionic main
+deb https://dl.bintray.com/allegro/debng bionic main
```

Comment:
debng - where does the name come from?

Reply:
deb - new generation

Reply:
The old repo "deb" has a deprecated layout so we have created a new one.